configuration/arm64/initialization.sh

```diff
@@ -38,17 +35,21 @@ fi
 
 #ARM initialization
 cd $directory
-sudo apt-get -y remove docker docker-engine docker.io containerd runc
-sudo apt-get -y install apt-transport-https ca-certificates curl gnupg-agent software-properties-common
-OS=$(lsb_release -i | awk 'FS=":" {print $3;}' | awk '{print tolower($0)}')
-arch=$(dpkg --print-architecture)
-curl -fsSL https://download.docker.com/linux/$OS/gpg | sudo apt-key add -
-sudo add-apt-repository "deb [arch=$arch] https://download.docker.com/linux/$OS $(lsb_release -cs) stable"
-sudo apt-get -y install docker-ce docker-ce-cli containerd.io docker-compose-plugin
-sudo apt -y install python3-pip
-sudo pip3 install docker-compose
-sudo apt update -y && sudo apt upgrade -y
-sudo docker-compose up -d
+for pkg in docker.io docker-doc docker-compose podman-docker containerd runc; do sudo apt-get remove $pkg; done
```

Comment:
Was getting errors, updated and tested using Docker's official Debian installation steps